cloud/amazon/ec2.py

```diff
@@ -499,6 +500,15 @@
     state: running
 
 #
+# Restart instances specified by tag
+#
+- local_action:
+    module: ec2
```

Comment:
Technically this also requires a `region: <somewhere>` option, otherwise the EC2 library can't connect. You can do this in a followup patch, no problem.